topostats/default_config.yaml

```diff
@@ -17,16 +17,18 @@ filter:
   gaussian_mode: nearest
 grains:
   run: true # Options : true, false
-  absolute_smallest_grain_size: 100
-  threshold_method: otsu # Options : otsu, std_dev, absolute
+  # Thresholding by height
+  threshold_method: std_dev # Options : otsu, std_dev, absolute
```

Comment:
I suspect the change in the default method will be the cause of tests failing.

Reply:
I changed the default back to otsu before I saw your update. I'll keep it this way for now, because the current dev branch has otsu as default for both the 'grains' and the 'filter' sections, and it'd make more sense for this pull request to be only about improving readability.
However, I do recall that we have decided in previous meetings that standard deviation should be the default for both sections, and this has been the case in previous versions of the dev branch when example.yaml was still used. I suspect the default was (intentionally or unintentionally) reverted to otsu during the removal of the duplicate config file. Is it worth creating another issue?

Reply:
Also, if we are certain that standard deviation is the preferable default method, are there any tests that need to be changed to reflect that?

Reply:
Thanks @Jean-Du 

Currently the regression tests under `tests/test_run_topostats.py` use the `threshold_method = "std_dev"` for both `Filter()` and `Grains()` processing.

The beauty of git is that we can go back through the [history of `default_config.yaml`]( https://github.com/AFM-SPM/TopoStats/commits/dev/topostats/default_config.yaml) and find where the change was introduced.

Doing so shows it was me who changed it, specifically so the tests could pass (see [this commit](https://github.com/AFM-SPM/TopoStats/commit/ec453ce3993dc78baab4b2acfd6bc447152a45e7#diff-969c222dca4d7238d7e79e8a6ac14b700a670ff1f85ec540ec4b7ed3d0ccb8e0)).

There is another way of doing this though and I shall now revert that in the configuration and modify the tests have used `otsu` in the past to use `std_dev`.

With regards to defaults across all values, not just threshold methods I think #302 which seeks to refine the schema would be a sufficient place to record this. It should probably be documented clearly why its the default and the implications of choosing other methods.